src/Exchange.js

```diff
@@ -36,17 +65,21 @@ export default function Exchange(props) {
   };
   const handleSwap = async (evt) => {
     evt.preventDefault();
-    await exchange(
+    console.log(`Exchanging ${inputAmount} ${JSON.stringify(inputToken)} for at least ${minimumOutputAmount} ${JSON.stringify(outputToken)}`)
```

Comment:
In theory, I agree, but I think at this stage, it may be really useful to have users open the console and send us the logs when there are issues. Really, even as it stabilizes, it doesn't bother me too much. Happy to remove them if/when you think, though!

Reply:
I've been removing them all before committing. Thats seems to be [the consensus](https://github.com/airbnb/javascript/blob/5124de23da0190b686850c9582e5a0960a607ae6/packages/eslint-config-airbnb-base/rules/errors.js#L26) in the JS community. Then when you do add a `console.log` it's easy to spot. Let's continue with that.